src/transform/convert/generate-static-fields.spec.js

```diff
@@ -27,10 +37,23 @@ function callback({
 }) {
   const inputData = new MarcRecord(getFixture('input.json'));
   const expectedResults = getFixture('output.json');
-  const result = handleProcess(functionToUse, inputData) // eslint-disable-line
+  const result = handleProcess(functionToUse, inputData); // eslint-disable-line
 
-  expect(result).to.eql(expectedResults);
+  if (result) {
+    expect(result).to.eql(expectedResults);
+    return;
+  }
 
+  /**
+  * Test process handling. Handles normal and error cases
+  * @date 12/04/2024 - 8:30:00 AM
+  *
+  * @param {string} functionToUse Name of function for testing
+  * @param {MarcRecord} inputData Input record
+  * @param {boolean} expectedError Is error expected from this test
+  * @param {string} expectedErrorStatus What is status of error that is expected
+  * @returns {object|false} Marc record result object or false
+  */
   function handleProcess(functionToUse, inputData) {
```

Comment:
Should this function take function parameters list as parameter? Otherwise custom parameters cannot be passed to called function from metadata.json definitions.

Reply:
As per slack conversation. In case it is needed, it can be added to this. (fixed error param passing to this)